packages/edit-post/src/components/header/header-toolbar/index.js

```diff
@@ -142,10 +142,17 @@ function HeaderToolbar() {
 					icon={ plus }
 					/* translators: button label text should, if possible, be under 16
 			characters. */
-					label={ _x(
-						'Add block',
-						'Generic label for block inserter button'
-					) }
+					label={
+						isInserterOpened
+							? _x(
+									'Close block inserter',
```

Comment:
Could we replace this with a single "Toggle block inserter" instead of the ternery? That way the label doesn't need to change, which I suspect might be better for screen readers.